app/code/community/Reverb/ReverbSync/Helper/Data.php

```diff
@@ -281,6 +281,7 @@ protected function _getCurlResource($url, $options_array = array())
         $options_array[CURLOPT_SSL_VERIFYPEER] = 0;
         $options_array[CURLOPT_HEADER] = 0;
         $options_array[CURLOPT_RETURNTRANSFER] = 1;
+        $options_array[CURLOPT_USERAGENT] = 'Reverb-Magento MagentoVersion=community-1.9';
```

Comment:
this is not supposed to be a hardcoded magento version. are we able to read the magento version off the instance?